server/utils/constants.js

```diff
@@ -0,0 +1,33 @@
+/*===============================================================================*/
+/*********************************************************************************/
+/**
+  * @fileOverview constants - File for defining constants
+  * @author T V Deekshith, venkatadeekshith@elear.solutions
+  * @copyright Copyright (c) 2021 Elear Solutions Tech Private Limited. All rights
+  * reserved.
+  * @license To any person (the "Recipient") obtaining a copy of this software and
+  * associated documentation files (the "Software"):
+  *
+  * All information contained in or disclosed by this software is confidential
+  * and proprietary information of Elear Solutions Tech Private Limited and all
+  * rights therein are expressly reserved. By accepting this material the
+  * recipient agrees that this material and the information contained therein is
+  * held in confidence and in trust and will NOT be used, copied, modified,
+  * merged, published, distributed, sublicensed, reproduced in whole or in part,
+  * nor its contents revealed in any manner to others without the express
+  * written permission of Elear Solutions Tech Private Limited.
+  */
+/*********************************************************************************/
+/*===============================================================================*/
+
+// service name to be used in logs
+export const SERVICE_NAME = "SAMPLETOKENSERVER";
+
+// COCO confedential client auth grantType
+export const COCO_SERVICE_APPLICATION_GRANT_TYPE = "client_credentials";
+
+// COCO refresh token auth grantType
+export const COCO_REFRESH_TOKEN_GRANT_TYPE = "refresh_token";
+
```

Comment:
- Create an Object called OAuthGrantTypes and inside that create keys like CLIENT_CREDENTIALS and REFRESH_TOKEN

Reply:
handled